docs/configuration-files.md

```diff
@@ -271,6 +272,117 @@ rules:
 
 This rule looks for usage of Django's [`FloatField`](https://docs.djangoproject.com/en/3.0/ref/models/fields/#django.db.models.FloatField) model when storing currency information. `FloatField` can lead to rounding errors and should be avoided in favor of [`DecimalField`](https://docs.djangoproject.com/en/3.0/ref/models/fields/#django.db.models.DecimalField) when dealing with currency. Here the `pattern-where-python` operator allows us to utilize the Python `in` statement to filter findings that look like currency.
 
+## Metavariable matching
+
+### Metavariables in logical inclusions
+
+Patterns' matched metavariable values are enforced to be identical when performing logical inclusion operations (`patterns`, `pattern-inside`) on matches.
```

Comment:
I'm not really sure what this is saying. Could we instead say something like "Metavariables persist across `pattern`, `patterns`, and `pattern-inside` operators."

Reply:
@mschwager would it be easiest if you just took a swing at the documentation? I'm definitely struggling to articulate this without getting way too technical.

Reply:
Just getting back to this, happy to iterate!